docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -156,3 +341,14 @@ subpackage :code:`fairlearn.metrics`.
    .. [#3] Agarwal, Beygelzimer, Dudik, Langford, Wallach `"A Reductions
       Approach to Fair Classification"
       <https://arxiv.org/pdf/1803.02453.pdf>`_, ICML, 2018.
+	  
+   .. [#4] Selbst, Andrew D. and Boyd, Danah and Friedler, Sorelle and Venkatasubramanian,
+      Suresh and Vertesi, Janet, "Fairness and Abstraction in Sociotechnical Systems" (August 23, 2018).
+      2019 ACM Conference on Fairness, Accountability, and Transparency (FAT*), 59-68, Available at
+      `SSRN: 	<https://ssrn.com/abstract=3265913>`_,
+   
+   .. [#5] Mark S. Ackerman. 2000. The intellectual challenge of CSCW: The gap between social requirements
```

Comment:
nit: can we add a link for these last two? Either from the venue/journal they were submitted to, or from ArXiv

Reply:
Added link

Reply:
The CSCW nerd in me is thrilled that Ackerman is cited in Fairlearn!! :)